devlib/module/cpuidle.py

```diff
@@ -61,13 +61,25 @@ def name(self):
     @property
     @memoized
     def latency(self):
+        """State exit latency in uS"""
         return self.get('latency')
 
     @property
     @memoized
     def power(self):
+        """State power usage in mW"""
```

Comment:
AFAICT this value is actually garbage on ARM. Wasn't sure whether to mention that here...